deploy/helm/charts/Chart.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: v1
```

```suggestion
apiVersion: v2
```

NDM uses v2 version.